src/components/Markdown.tsx

```diff
@@ -163,73 +182,91 @@ const renderCardrow: React.FC<RenderCardrowProps> = (node) => (
 interface RenderH1Props {
   children: ReactNode;
 }
-const renderH1: React.FC<RenderH1Props> = (node) => (
-  <Text xxxxl semibold className="mb-4">
-    {node.children}
-  </Text>
-);
+const renderH1: React.FC<RenderH1Props> = (node) => {
```

Comment:
Feels like there should be some way to have a single version of render header that takes only the size (eg xxxxl) as a parameter, but I couldn't figure out the currying